.github/workflows/ci.yml

```diff
@@ -277,10 +304,26 @@ jobs:
           path: ${{ env.package_path }}
           retention-days: 7
 
+      - name: Upload binaries for .NET Bindings NuGet package
+        # "There can be only one."
+        if: matrix.name == 'Ubuntu 22.04 gcc-12 Release'
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ env.linux_x64_artifact }}
+          path: |
+            ${{ env.build_path }}/bin/*.so
+            !${{ env.build_path }}/**/opendaq.cpython*
+            !${{ env.build_path }}/**/*_test_dll.so
+          # the exclusion filter does not work and the online-help does not explain
+          # most probably a pattern *xxx.yy does not work, only x*
+          #if-no-files-found: error # 'warn' or 'ignore' are also available, defaults to `warn`
+          retention-days: 1
+
   build_linux_centos:
     runs-on: ubuntu-latest
     name: ${{ matrix.name }}
-    if: '! github.event.pull_request.draft'
```

Comment:
Is it not working anymore?

Reply:
It is, but for anyone new to yaml there are too many "valid syntaxes" for these `if` cases. I just decided to make it more clear in using the common syntax `${{ ! ... }}` and not any specials `! ...`. 
Effectively this has cost me more than one hour to figure out the syntax (I was using the not `!` operator without the quotes or braces and wondered why it failed). Since the braces are used almost everywhere else I decided to replace it for clarity, even if it looks more complex.